analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -24,6 +27,64 @@
 LOG = logger.get_logger('system')
 
 
+def get_diagtool_bin(env=None):
+    """
+    Return full path of diagtool.
+
+    Select clang binary, check for a 'diagtool' binary next to the selected
+    clang binary and return full path of this binary if it exists.
+    """
+    context = analyzer_context.get_context()
+    analyzer_binary = context.analyzer_binaries.get(ClangSA.ANALYZER_NAME)
+
+    clang_bin = find_executable(analyzer_binary,
```

Comment:
It is possible that the configured clang binary is not in the PATH?
For example I've installed clang6 from the ubuntu package which is in the PATH, but I've built clang8 from source which I set in the config file. In that case will this find the right diagtool next to the clang8 build?